wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md

```diff
@@ -138,7 +138,7 @@ Until lazer, sliders have only required the accuracy of a 50/MEH judgement to re
 
 Missing a slider head (either by not hitting it or hitting it during its miss window) would previously break combo but not cause a MISS judgement, and a judgement could still be received for the missed slider by completing the rest of it. This allowed players to get scores with low max combo while technically having no misses.
 
-In lazer, not hitting the slider head will give a MISS judgement for the whole slider. After missing a slider head, combo, score, and accuracy can still be gained from slider ticks and repeats, while only score and accuracy can still be gained from the slider end.
+In lazer, not hitting the slider head will give a MISS judgement for the whole slider. After missing a slider head, combo, score, and accuracy can still be gained from slider ticks, repeats and ends.
```

Comment:
```suggestion
In lazer, not hitting the slider head will give a MISS judgement for the whole slider. After missing a slider head; combo, score and accuracy can still be gained from slider ticks, repeats and ends.
```

Reply:
that's not really how semicolon works. i see what you were going for but u probs have to just rewrite the sentence to have less ambiguity in punctuation there

Reply:
okay